frontend/src/components/pages/CourseOverview.tsx

```diff
@@ -214,6 +212,14 @@ const CourseOverview = (): React.ReactElement => {
           </Box>
         </Flex>
       </Flex>
+      <VStack w="calc(100% - 28rem)" py="2rem" px="7.5rem" align="left">
+        <Text variant="heading">Course Content</Text>
+        <ModuleDropdowns
```

Comment:
Can you double check all the functionality works when the user is logged out? Victor had run into some weird edge cases there.